docs/content/docs/test/Runner.md

```diff
@@ -187,8 +187,70 @@ extensions.getByType<GradleInstrumentationPluginConfiguration>().apply {
 }
 ```
 
+### Change applied filter without changing `build.gradle.kts`
+
+1. Add custom gradle property for filter name to gradle.properties file
+
+```properties
+filterName="filterName"
+```
+
+2. Let plugin configuration depends on `property`
```

Comment:
```suggestion
2. Use the property to configure a plugin
```

- We use backticks use only for [exact names](https://avito-tech.github.io/avito-android/docs/contributing/docs/#%D0%B2%D1%8B%D0%B4%D0%B5%D0%BB%D0%B5%D0%BD%D0%B8%D0%B5-%D1%82%D0%B5%D1%80%D0%BC%D0%B8%D0%BD%D0%BE%D0%B2)
- "depends on" can be interpreted as grade `dependsOn`